### PR TITLE
Added ChoiceBoard text in weekplan

### LINKED ISSUE #534: Missing pictogram text under choiceboard activities causes inconsistent activity sizes in weekplan
**Describe the bug**
There is no pictogram text under choiceboard activities resulting in the activities not having the same size on the weekplan screen.
image

**Expected behavior**
Some standard pictogram text like ChoiceBoard should probably be added.
Ask @aau-giraf/sw613f20 for expected behavior

### PR DESCRIPTION
**Pull request by Simon97**
fixes #534

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw616f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw617f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

**Comment by Semicolon1071** [comment]
Den går ikke igennem tests?

**Review by Torpstitution — DISMISSED** [review]

File: lib/widgets/pictogram_text.dart [hunk 1]
```diff
@@ -46,9 +45,13 @@ class PictogramText extends StatelessWidget {
...
                   final SettingsModel settings = settingsSnapshot.data;
                   final bool pictogramTextIsEnabled = settings.pictogramText;
-                  if (_isGuardianMode(weekMode) || pictogramTextIsEnabled) {
+                  if (_activity.isChoiceBoard == false &&
+                      (_isGuardianMode(weekMode) || pictogramTextIsEnabled)) {
                     final String pictogramText = _pictogram.title.toUpperCase();
                     return _buildPictogramText(context, pictogramText);
+                  } else if (_activity.isChoiceBoard &&
+                      (_isGuardianMode(weekMode) || pictogramTextIsEnabled)) {
+                    return _buildPictogramText(context, 'ChoiceBoard');
```
Comment (giano574) [inline lib/widgets/pictogram_text.dart]: Lidt lettere at læse:
```suggestion
                  if (_isGuardianMode(weekMode) || pictogramTextIsEnabled) {
                    if (_activity.isChoiceBoard) {
                      return _buildPictogramText(context, 'ChoiceBoard');
                    } else {
                      final String pictogramText = _pictogram.title.toUpperCase();
                      return _buildPictogramText(context, pictogramText);
                    }
                  }
```

File: lib/widgets/pictogram_text.dart [hunk 2]
```diff
@@ -47,8 +46,13 @@ class PictogramText extends StatelessWidget {
...
-                    final String pictogramText = _pictogram.title.toUpperCase();
-                    return _buildPictogramText(context, pictogramText);
+                    if (_activity.isChoiceBoard) {
+                      return _buildPictogramText(context, 'ChoiceBoard');
+                    } else {
+                      final String pictogramText = _pictogram.title.toUpperCase();
+                      return _buildPictogramText(context, pictogramText);
+                    }
+                  }
                   }
```
Comment (giano574) [inline lib/widgets/pictogram_text.dart]:
```suggestion

```

**Review by giano574 — APPROVED** [review]

**Review by Torpstitution — APPROVED** [review]

**Pull request merged**